Create revision branch for Q4 2021 (#17)
bonusly/un-handbook: README.md

## What We Believe
#### At Bonusly, we believe...
 * Success in business is determined by **people** **working on teams**.

#### This belief inspires us to...
 * Maximize the conditions for people and teams to sustainably do their best work over the long run:
   * Purpose - the sense that your job is important and you are having an impact
   * Progress - the opportunity to develop your skills and grow as a person, as well as test yourself against new and interesting challenges
   * Belonging - the feeling that you matter and are valued, combined with the bond you forge with colleagues as your strive toward a common goals

#### We work toward a future in which...
 * Every organization achieves success by developing purposeful, inclusive, and high-achieving teams.

## Values

> These values are what we value at Bonusly. They aren’t just nice-sounding words on a poster somewhere, but a succinct description of our culture and what it’s like to work here. We use these values to guide our day-to-day activities and decision-making, from how we respond to customer emails to the roles we hire for to the features we build.

Reviewer: Something about having values, values, value all within a few words is hurting my brain a bit, and not sure if we need them given the subheading of Values right there.

```suggestion
They aren't just nice-sounding words on a poster somewhere, but a succinct description of our culture and what it’s like to work here. We use these values to guide our day-to-day activities and decision-making, from how we respond to customer emails to the roles we hire for to the features we build.
```

### Seek and embrace diversity

Embodying this value means:

> * Actively learn about your own privileges

Reviewer: Love this change

Our industry has [well-documented](http://www.theverge.com/2015/8/20/9179853/tech-diversity-scorecard-apple-google-microsoft-facebook-intel-twitter-amazon) [problems](http://wpusa.org/WPUSA_TechsDiversityProblem.pdf) when it comes to [workplace diversity](http://techcrunch.com/2015/07/28/what-a-truly-diverse-tech-company-would-look-like/). This fact makes it easy for tech companies to skew everything from their employment policies to the features they develop toward the needs of the dominant group - which is both a disservice to our customers and reinforces the lack of diversity in our industry.

Our dedication to diversity goes beyond simply putting equal opportunity copy on our job descriptions. We use tools (like [Mathison](https://www.mathison.io/platform)) to eliminate gender bias from the language of our job postings, and actively seek out applicants from diverse backgrounds. We all have [unconscious biases](https://www.youtube.com/watch?v=nLjFTHTgEVU), so we strive to build bias-mitigation into our training, processes, and systems. And in the spirit of measuring results, we track and share our diversity numbers on a quarterly basis.

> On a product level, we strive to build a software tools that are inclusive and welcoming to all, and will hold ourselves accountable for how our product is used by our customers.

Reviewer: Not something you changed in this pr, but worded real weird as it is right now

```suggestion
On a product level, we strive to build software that is inclusive and welcoming to all, and will hold ourselves accountable for how our product is used by our customers.
```

Can you truly take unlimited vacation? Of course not. You can't expect to take 6 months of paid vacation per year. But, there is also not a hard limit on the number of days off per year that you can take. Between longer vacations and random days off here and there, a typical employee should take enough time off that they are able to sustainably perform their job function at a high level.

> Bonusly observes all [US federal holidays](https://www.federalpay.org/holidays), [Native American Heritage Day](https://en.wikipedia.org/wiki/Native_American_Heritage_Day) (Friday after Thanksgiving), [Juneteenth](https://juneteenth.com/history/), and [Indigenous Peoples Day](https://www.smithsonianmag.com/blogs/national-museum-american-indian/2019/10/11/indigenous-peoples-day-2019/) as paid holidays.

Reviewer: [Peoples -> Peoples'](https://www.nytimes.com/2021/10/11/us/indigenous-peoples-day.html)

```suggestion
Bonusly observes all [US federal holidays](https://www.federalpay.org/holidays), [Native American Heritage Day](https://en.wikipedia.org/wiki/Native_American_Heritage_Day) (Friday after Thanksgiving), [Juneteenth](https://juneteenth.com/history/), and [Indigenous Peoples' Day](https://www.smithsonianmag.com/blogs/national-museum-american-indian/2019/10/11/indigenous-peoples-day-2019/) as paid holidays.
```
